regenerate_index_html.py

```diff
@@ -24,7 +31,16 @@
             readme_filename = f
         else:
             filenames.append(f)
-    links.append('</ul><ul>')
+
+    title = '<h3>%s</h3>' % (dirname)
```

Comment:
```suggestion
    title = '%s' % (dirname)
```
<h3> wrapping done below